pkg/rdkafka/composer.json

```diff
@@ -7,7 +7,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.1.3",
-        "ext-rdkafka": "^3.0.3",
+        "ext-rdkafka": "^3.1.0",
```

Comment:
> And the inclusion of that code for `produce` fallback means that version from `^3.0.3` to `^3.1.0` is no longer required.

Reply:
Oops, forgot about it. Of course you're right. Reverted it.